Spring-2019/linked_lists/SOLUTIONS_2019Spring.md

```diff
@@ -89,20 +103,67 @@ TODO :bug:
 
 TODO :bug:
 
-## 3. PROBLEM 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
+## 3. Rotate List
 
-TODO :bug:
+Source: [CrazyForCode](http://www.crazyforcode.com/rotate-linked-list-k-nodes/)
 
 #### Optimal Solution
 
-TODO :bug:
-
-#### Testing The Solutions OR Driver For Solution
-
-TODO :bug:
-
-
+_Note: An important question to ask with this question is whether `k` is 
+guaranteed to be smaller than the length of the list. This implementation 
+assumes this is true._
```

Comment:
We should include this as an assumption in the docs problem

Reply:
fixed in 775a59533001aa2787e743085c5b3c7c1ae69c34